collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_network.py

```diff
@@ -204,6 +204,7 @@ def update_interface(module, array, interface):
             ]:
                 module.fail_json(msg="Gateway and subnet are not compatible.")
         address = str(module.params["address"].split("/", 1)[0])
+        ip_version = str(IPAddress(address).version)
```

Comment:
This needs to be moved in 4 spaces so that it is set even if the `address` parameter is not set, otherwise your change on line 231 will not have a value set for `ip_version`